scripts/qa_crowbarsetup.sh

```diff
@@ -2335,7 +2335,10 @@ function oncontroller_testsetup()
         crudini --set /etc/tempest/tempest.conf orchestration image_ref $imageid
         pushd /var/lib/openstack-tempest-test
         echo 1 > /proc/sys/kernel/sysrq
-        ./run_tempest.sh -N $tempestoptions 2>&1 | tee tempest.log
+        if [ ! -d .testrepository ]; then
+            testr init
+        fi
+        testr run $tempestoptions | tee tempest.log | subunit-2to1 | tools/colorizer.py
```

Comment:
this breaks with cloud version where tempest is installed on sle11, as testr is not upstream supported for python 2.6. 

IMHO it would be easier to add a small patch to run_tempest.sh that logs the subunit output to a separate file (this way you're also not changing the format of tempest.log)